docs: update package manager in readme (#713)
nuxt/bridge: README.md

- Clone repository
- Ensure you have the latest LTS version of Node.js installed

> - Install dependencies with `npx pnpm install`
> - Run `npx pnpm dev:prepare` to activate passive development
> - Open playground with `npx pnpm dev`

Reviewer: I think we can require corepack use.
```suggestion
- Enable [Corepack](https://github.com/nodejs/corepack) using `corepack enable` to enable `pnpm` and `yarn`
- Install dependencies with `pnpm install`
- Run `pnpm dev:prepare` to activate passive development
- Open playground with `pnpm dev`
```

### Upgrade to the latest Nuxt 2

> Make sure your dev server (`nuxt dev`) isn't running, remove any package lock files (`package-lock.json`, `yarn.lock` and `pnpm-lock`), and install the latest `nuxt` version:

Reviewer: ```suggestion
Make sure your dev server (`nuxt dev`) isn't running, remove any package lock files (`package-lock.json` and `pnpm-lock.yaml`), and install the latest `nuxt` version:
```

### Upgrade to the latest Nuxt 2

> Make sure your dev server (`nuxt dev`) isn't running, remove any package lock files (`package-lock.json` and `pnpm-lock.yaml`), and install the latest `nuxt` version:

Reviewer: ```suggestion
Make sure your dev server (`nuxt dev`) isn't running, remove any package lock files (`package-lock.json`, `yarn.lock` or `pnpm-lock.yaml`), and install the latest `nuxt` version:
```
